README.md

```diff
@@ -60,6 +61,13 @@ For example: a button must always end with `Button`.
 There is no limit for the length of a name, so prefer a long name which is clear and descriptive
 than a short name which is not clear.
 
+#### Context
+A name should make sense within it's context and should not have unnecessary information for that
```

Comment:
A name should make sense within **it's** context

A name should make sense within **its** context

Reply:
Fixed